vmm/sandbox/src/stratovirt/mod.rs

```diff
@@ -450,11 +458,14 @@ impl StratoVirtVM {
         &mut self,
         mut device: T,
     ) -> Result<()> {
-        // get the empty slot index
```

Comment:
Now `microVM` support a new virtual bus type`mmio`, so `attatch_to_pcie_rootbus` func name is not accurate, maybe `attach_to_bus` is more clear.